package.json

```diff
@@ -0,0 +1,62 @@
+{
+    "name": "jellyfin-web",
+    "description": "",
+    "version": "10.0.1",
```

Comment:
Use `0.0.0` until a versioning scheme is defined in https://github.com/jellyfin/jellyfin-web/issues/3